clang/test/Driver/clang-offload-bundler-bc-archive-support.cpp

```diff
@@ -0,0 +1,121 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Make bundled object with targets:
+// sycl-spir64-unknown-unknown
+// host-x86_64-unknown-linux-gnu
+// RUN: %clangxx -fsycl -c %s -o %t_bundled.o
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Make three distinct BC files
+// RUN: %clangxx -fsycl -fsycl-device-only -DTYPE1 %s -o %t1.bc
+// RUN: %clangxx -fsycl -fsycl-device-only -DTYPE2 %s -o %t2.bc
+// RUN: %clangxx -fsycl -fsycl-device-only -DTYPE3 %s -o %t3.bc
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Bundle BC files to different targets:
+// host-spir64-unknown-unknown
+// host-spir64_gen
+// host-spir64_x86_64
+// RUN: clang-offload-bundler -type=bc -targets=host-spir64-unknown-unknown -input=%t1.bc  -output=%t1_bundled.bc
+// RUN: clang-offload-bundler -type=bc -targets=host-spir64_gen             -input=%t2.bc  -output=%t2_bundled.bc
+// RUN: clang-offload-bundler -type=bc -targets=host-spir64_x86_64          -input=%t3.bc  -output=%t3_bundled.bc
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Make archive with bundled BC and o files
+// RUN: rm -f %t_bundled.a
+// RUN: ar cr %t_bundled.a %t1_bundled.bc %t2_bundled.bc %t3_bundled.bc %t_bundled.o

```

Comment:
Please try using llvm-ar instead. Thanks

Reply:
Changed to use llvm-ar